Prepare for v1.0.0 release (#31)
leancodepl/bloc_presentation: packages/bloc_presentation/CHANGELOG.md

# 1.0.0

> - **BREAKING**: Remove `useBlocPresentationListener` hook and replace it with `BlocPresentationListener` widget.

Reviewer: It wasn't replaced, the widget was already there

Reviewer 2: Actually, replace it with `useOnStreamChange` from `flutter_hooks`. But this is changelog. We don't say to replace, we say it's been replaced. We can form a next sentence suggesting to replace

Author: Right. It was totally reimplemented so I assumed wrongly it's something new

> - **BREAKING**: Removed `useBlocPresentationListener` hook and reimplemented `BlocPresentationListener` widget because of `flutter_hooks` dependencies removal.

Reviewer 2: ```suggestion
- **BREAKING**: Removed `useBlocPresentationListener` hook. Use `useOnStreamChanged` from `flutter_hooks` instead.
- Removed dependency on `flutter_hooks`.
```
